chrome/skin/default/zotero/overlay.css

```diff
@@ -237,6 +237,19 @@
 	margin-inline-end: 2px;
 }
 
+.zotero-tb-button[type="menu"] > .toolbarbutton-menu-dropmarker {
+	-moz-default-appearance: none;
+	background-image: url("chrome://zotero/skin/searchbar-dropmarker.png");
+	width: 7px;
+	height: 4px;
+	background-repeat: no-repeat;
+	background-size: 100%;
+}
```

Comment:
The above is not needed on Linux. The `-moz-default-appearance: toolbarbutton-dropdown` set in toolbarbutton.css is correct for Linux, and none of the other css properties affect this element when the appearance prop is set.